Gemfile

```diff
@@ -61,3 +61,5 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 gem 'rails-controller-testing'
 
```

Comment:
- [ ] Please update your Pull Request description which should explain the changes you made on this branch